Source/WebKit/NetworkProcess/Downloads/cocoa/DownloadCocoa.mm

```diff
@@ -64,15 +65,31 @@
     m_downloadTask.get()._pathToDownloadTaskFile = path;
 
     [m_downloadTask resume];
+
+#if HAVE(MODERN_DOWNLOADPROGRESS)
+    NSString *placeholderURLString = [dictionary objectForKey:@"ResumePlaceholderURL"];
+    if (placeholderURLString) {
```

Comment:
`if (RetainPtr placeholderURLString = [dictionary objectForKey:@"ResumePlaceholderURL"]) { `

Reply:
Ah, good point! I will address this and the other comments in the upcoming patch.

Thanks for reviewing!